x/bond/client/cli/tx.go

```diff
@@ -0,0 +1,125 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/spf13/cobra"
+	"github.com/tharsis/ethermint/server/flags"
+	"github.com/tharsis/ethermint/x/bond/types"
+)
+
+// NewTxCmd returns a root CLI command handler for all x/bond transaction commands.
+func NewTxCmd() *cobra.Command {
```

Comment:
How did we test that everything is working? Can we add a comment to the PR with commands and output for all bond CLI commands.

Reply:
Okay i will add commands and their results as comment to this pull request